docs/log_format.md

```diff
@@ -0,0 +1,58 @@
+# Log Format
+
+## Log Line Prefix
+
+Shadow produces simulator log messages in the following format:
+
+```text
+real-time [thread-id] virtual-time [logdomain-loglevel] [hostname~ip] [function-name] MESSAGE
+```
+
+**FIXME**: The `real-time` and `virtual-time` use different time units. Is this correct, or should we change this in Shadow?
+
+- `real-time`:  
+the wall clock time since the start of the experiment, represented as `hours:minutes:seconds:microseconds`
+- `thread-id`:  
+the ID of the worker thread that generated the message
+- `virtual-time`:  
+the simulated time since the start of the experiment, represented as `hours:minutes:seconds:nanoseconds`
+- `logdomain`:  
+either `shadow` or the name of one of the applications as specified in the _id_ tag of the _plugin_ element in the XML file (e.g., `tgen`, `tor`, `bitcoin`)
+- `loglevel`:  
+one of `error` < `critical` < `warning` < `message` < `info` < `debug`, in that order
```

Comment:
I think this is now `ERROR` < `WARN` < `INFO` < `DEBUG` < `TRACE`

Reply:
Fixed.